src/app/views/sidebar/resource-explorer/resource-explorer.utils.ts

```diff
@@ -94,14 +91,14 @@ export function createResourcesList(
       availableMethods
     ).sort(sortResourceLinks); // show graph functions at the top
 
-    // if segment has one method only and no children, do not make segment a node
-    if (availableMethods.length === 1 && versionedChildren.length === 0) {
+    // if segment has one method only and at most 1 child, do not make segment a node
+    if (availableMethods.length === 1 && versionedChildren.length <= 1 ) {
       paths = [...paths, segment];
       method = availableMethods[0];
     }
     const type = getLinkType({ ...info, links: versionedChildren });
     const enclosedCounter =
-      versionedChildren && versionedChildren.length > 0
+      versionedChildren && versionedChildren.length > 1
```

Comment:
This line hinders the counter displaying when a user searches for a resource and the parent node has only 1 child that matches the search term. 
Leaving it as it was ensures the counter always displays
```suggestion
      versionedChildren && versionedChildren.length > 0
```